client/landing/jetpack-cloud/routes.js

```diff
@@ -6,24 +6,12 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import {
-	clientRender,
-	jetpackCloud,
-	jetpackBackups,
-	jetpackScan,
-	makeLayout,
-	setupSidebar,
-} from './controller';
-import { normalize } from 'lib/route';
+import { jetpackCloud } from './controller';
+import { navigation } from 'my-sites/controller';
+import { makeLayout, render as clientRender } from 'controller';
 
 const router = () => {
-	page( '*', normalize );
```

Comment:
So we're no longer going to handle 404s on the server side?

Reply:
They are handled on the server side. I don't understand the question?